### PR TITLE
feat(sdk)!: move v1 to deprecated folder

### PR DESCRIPTION
**Pull request by ji-yaqi**

### CODE REVIEW

File: sdk/python/kfp/v2/compiler_cli_tests/test_data/lightweight_python_functions_v2_pipeline.json [hunk 1]
```diff
@@ -245,5 +245,5 @@
     }
   },
   "schemaVersion": "2.1.0",
-  "sdkVersion": "kfp-1.8.9"
+  "sdkVersion": "kfp-1.8.11"
```
Comment (chensun) [inline sdk/python/kfp/v2/compiler_cli_tests/test_data/lightweight_python_functions_v2_pipeline.json:248]: nit: you don't need to update the v2 golden files, they are unrelated to this change. Leaving them alone can give you a smaller diff.

**Review by chensun — COMMENTED** [review]
/lgtm

Feel free to merge by yourself upon fixing the tests.

File: samples/core/parameterized_tfx_oss/parameterized_tfx_oss.py [hunk 2]
```diff
@@ -13,168 +13,170 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import json
-import os
-
-import kfp
-import kfp.onprem
```
Comment (chensun) [inline samples/core/parameterized_tfx_oss/parameterized_tfx_oss.py:20]: We should make this sample work, as it's one of the pre-load sample pipelines shipped with KFP release. Removing this sample is actually the cause for the backend e2e test failure.
What was the issue if we use `kfp.deprecated` here?

**Comment by droctothorpe** [comment]
I'm preparing a PR against the `Client` class in what's now `deprecated/_client.py`. Are there plans to replace that client in v2? Is there a timeline for that? If so, contributing to the deprecated client might not be an effective use of my time.

**Pull request merged**